tests/tasks/setup/kit/validate-dataplane.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: validate-data-plane
+  namespace: tekton-pipelines
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/categories: Kubernetes
+    tekton.dev/tags: CLI, kubectl
+    tekton.dev/displayName: "kubernetes actions"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: |
+    Wait and validate the data plane in a guest cluster.
+  params:
+  - name: desired-nodes
+    description: The desired number of nodes in the cluster.
+  workspaces:
+  - name: config
+    mountPath: /config/
+  stepTemplate:
+    env:
+    - name: KUBECONFIG
+      value: /config/kubeconfig
+  steps:
+  - name: setup-data-plane
```

Comment:
step name doesn't make sense for the task. Also, can we not have this as separate task and add it as a step to setup task ? Or do we have any reason to separate out this as it's own Task ?